src/platforms/apple/common/performance/instrumentation/swiftui-instrumentation.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: SwiftUI Instrumentation
+sidebar_order: 12
+description: "Learn how to monitor the performance of your SwiftUI views."
+---
+
+<Include name="beta-note.mdx" />
+
+You can monitor the performance of your views in a SwiftUI project with the SentrySwiftUI library, which you can install via <PlatformLink to="/install/swift-package-manager/">SPM (Swift package manager)</PlatformLink> or CocoaPods:
+
+```ruby
+platform :ios, '13.0'
+use_frameworks! # This is important
+
+target 'YourApp' do
+  pod 'SentrySwiftUI'
+end
+```
```

Comment:
`h`: As long as we don't fix https://github.com/getsentry/sentry-cocoa/issues/2581, we should only mention SPM here, please.

Reply:
We need to fix it before merging this documents